Euclidean traveling salesperson.md

```diff
@@ -0,0 +1,40 @@
+# Summary
+This task involves finding the shortest path among a number of “cities” on a map to mimic complex problem solving in which each aspect of a problem depends on other aspects of that problem.
+
+# References
+> Main paper: https://www.pnas.org/content/pnas/115/35/8734.full.pdf
+> > Supporting paper: https://www.pnas.org/content/pnas/suppl/2018/08/09/1802407115.DCSupplemental/pnas.1802407115.sapp.pdf 
+
```

Comment:
Data for replication: https://dataverse.harvard.edu/dataset.xhtmlpersistentId=doi:10.7910/DVN/TSSQLY&version=2.0